pxr/imaging/hio/glslfxConfig.cpp

```diff
@@ -546,7 +546,7 @@ HioGlslfxConfig::_GetTextures(VtDictionary const & dict,
     }
 
     const VtDictionary& texturesDict = textures.UncheckedGet<VtDictionary>();
-    for (const std::pair<std::string, VtValue>& p : texturesDict) {
+    for (const std::pair<std::string, VtValue> p : texturesDict) {
```

Comment:
You should be able to avoid the copy by making `p` `const auto&` or `const std::pair<const std::string, VtValue>&`.